client/app/scripts/components/time-travel-wrapper.js

```diff
@@ -78,12 +78,10 @@ function mapStateToProps(state) {
     topologyViewMode: state.get('topologyViewMode'),
     currentTopology: state.get('currentTopology'),
     timestamp: state.get('pausedAt'),
-    // Used only to trigger recalculations on window resize.
-    viewportWidth: state.getIn(['viewport', 'width']),
```

Comment:
Don't need this anymore?

Reply:
No, the TimeTravel in `ui-components` has its own resize listener.